Package.swift

```diff
@@ -4,6 +4,6 @@ let package = Package(
     name: "VaporSQLite",
     dependencies: [
    		.Package(url: "https://github.com/vapor/sqlite-driver.git", majorVersion: 1),
-   		.Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1),
+   		.Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor:1),
```

Comment:
I think for this, we should probably do the `Version(1,1,0)..<Version(2,0,0)` thing for greater compatibility